roles/install_dbserver/tasks/main.yml

```diff
@@ -2,3 +2,33 @@
 # Entry point of the install_dbserver role
 - name: Include the install_dbserver.yml
   ansible.builtin.include_tasks: install_dbserver.yml
+
+- name: Set Postgres Tuner package prefix (PG)
+  ansible.builtin.set_fact:
+    pg_tuner_prefix: "{{ pg_type | lower }}"
+  when: >-
+    pg_type == "PG"
+    and enable_edb_repo|bool
+    and install_edb_postgres_tuner|bool
+    and pg_version|int >= 11
+
+- name: Set Postgres Tuner package prefix (EPAS)
+  ansible.builtin.set_fact:
+    pg_tuner_prefix: "as"
+  when: >-
+    pg_type == "EPAS"
+    and enable_edb_repo|bool
+    and install_edb_postgres_tuner|bool
+    and pg_version|int >= 11
+
+- name: Install EDB Postgres Tuner
+  ansible.builtin.package:
+    name:
+      - "edb-{{ pg_type | lower }}{{ pg_version }}-\
+        postgres-tuner{{ pg_tuner_version }}"
+    state: present
+  become: true
+  when: >-
+    enable_edb_repo|bool
+    and install_edb_postgres_tuner|bool
+    and pg_version|int >= 11
```

Comment:
You could replace all the above code by declaring a variable in a default/main.yml file of the role something like given below:
`edb_tuner_package: "edb-{{ 'as' if pg_type = 'EPAS' else 'pg' }}{{ pg_version }}-postgres-tuner{{ pg_tuner_version }}"`
And use the following tasks:
```
- name: Install EDB Postgres Tuner
   ansible.builtin.package:
     name: "{{ edb_tuner_package }}"
     state: present
   become: true
   when: >-
     enable_edb_repo|bool
     and install_edb_postgres_tuner|bool
     and pg_version|int >= 11
```

Reply:
Ok, will do.